frameworks/digital-outcomes-and-specialists/questions/brief-responses/legacyAvailabilityOutcomes.yml

```diff
@@ -1,4 +1,5 @@
 name: Date the supplier can start work
+short_name: Earliest start date
```

Comment:
We shouldn't need to make any changes to the legacy questions as we are not changing anything with the CSV.